pkg/schedule/operator_controller.go

```diff
@@ -711,20 +726,18 @@ func (oc *OperatorController) OperatorCount(kind operator.OpKind) uint64 {
 
 // GetOpInfluence gets OpInfluence.
 func (oc *OperatorController) GetOpInfluence(cluster Cluster) operator.OpInfluence {
-	influence := operator.OpInfluence{
-		StoresInfluence: make(map[uint64]*operator.StoreInfluence),
-	}
+	influence := operator.NewOpInfluence()
 	oc.RLock()
 	defer oc.RUnlock()
 	for _, op := range oc.operators {
 		if !op.CheckTimeout() && !op.CheckSuccess() {
 			region := cluster.GetRegion(op.RegionID())
 			if region != nil {
-				op.UnfinishedInfluence(influence, region)
+				op.UnfinishedInfluence(*influence, region)
```

Comment:
Can we change the `opInfluence` as pointer in `UnfinishedInfluence`？

Reply:
I will revert this change